pages/mfbarrera/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Hotels</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <section>
+    <div class="container">
+      <div class="background">
+        <div class="carousel__header">OUR HOTELS</div>
```

Comment:
Usually, elements should be inside of its block, `carousel` should be the father of `carousel__header`

Reply:
Fixed. it is a header itself.